docs/conf/prague/2020/schedule.rst

```diff
@@ -11,9 +11,14 @@ Each year we organize a wide range of events so that people can come together, c
     :depth: 1
     :backlinks: none
 
+
+{% if flaghashike or flaghasboat %}
```

Comment:
For some reason I can't insert a suggestion directly to the boat ride text, but we need to match it with what it says on the home page of the conference (i.e. "We invite you to do a boat ride or walk on your own)

Reply:
Done.